src/batch.rs

```diff
@@ -9,12 +9,7 @@
 
 //! Batch signature verification.
 
-#[cfg(feature = "alloc")]
-extern crate alloc;
-#[cfg(feature = "alloc")]
 use alloc::vec::Vec;
-#[cfg(all(not(feature = "alloc"), feature = "std"))]
-use std::vec::Vec;
```

Comment:
Changed these to use `alloc` unconditionally.

`alloc` is already a hard requirement of batch verification.